third_party/tsl/tsl/profiler/lib/nvtx_utils.h

```diff
@@ -34,6 +34,12 @@ using ProfilerDomainHandle = ProfilerDomain*;
 // Get the "TSL" domain if NVTX profiling is enabled, otherwise null
 ProfilerDomainHandle DefaultProfilerDomain();
 
+// Assign a human-readable name to the current thread
```

Comment:
Define the interface to take const std::string& and call c_str() within the implementation.

Reply:
Done